    body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"f9d14fe7-6121-4b54-a213-9831bbd1c74f"}'

}

[NOTION DEBUG - notion.pages.create-payload]: {

  "parent": {

    "database_id": "21f2c4e3-db09-811e-bba7-d0b87a201f15"

  },

  "properties": {

    "Quote": {

      "title": [

        {

          "text": {

            "content": ">Interest in traumatic bereavement niche</indicator>\n  <indicator is_question_relevant="

          }

        }

      ]

    },

    "Category": {

      "select": {

        "name": "extracted_text"

      }

    },

    "Sentiment": {

      "select": {

        "name": "neutral"

      }

    },

    "Theme": {

      "select": {

        "name": "general"

      }

    },

    "Reddit Link": {

      "url": "https://www.reddit.com/r/therapists/comments/1fo6oqr/choosing_trainings_for_development_and_the/";

    },

    "Relevance Score": {

      "number": 0

    },

    "Date Added": {

      "date": {

        "start": "2025-06-27T00:36:19.038294+00:00"

      }

    },

          }

    },

    "Post ID": {

      "rich_text": [

        {

          "text": {

            "content": "0661d10c-911d-4871-95fc-a31692fbf79a"

          }

        }

      ]

    }

  }

}

[NOTION DEBUG - notion.pages.create-payload]: {

  "parent": {

    "database_id": "21f2c4e3-db09-811e-bba7-d0b87a201f15"

  },

  "properties": {

    "Quote": {

      "title": [

        {

          "text": {

            "content": ">Licensed therapist navigating job options after layoff</indicator>\n  <indicator is_question_relevant="

          }

        }

      ]

    },

    "Category": {

      "select": {

        "name": "extracted_text"

      }

    },

    "Sentiment": {

      "select": {

        "name": "neutral"

      }

    },

    "Theme": {

      "select": {

        "name": "general"

      }

    },

    "Reddit Link": {

      "url": "https://www.reddit.com/r/therapists/comments/1cne5bn/feeling_desperate_for_a_job/";

    },

    "Relevance Score": {

      "number": 0

    },

    "Date Added": {

      "date": {

        "start": "2025-06-27T00:36:21.184545+00:00"

      }

    },

    "Post ID": {

}

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"3f10699a-f191-4d54-89be-f98c27a05670"}'

}

@notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"72a8df72-cb2a-404d-b16f-9acd768e0298"}'

}

@notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"2efa5636-aad3-42ce-9516-9fd01aeeda95"}'

}

@notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"7dc2e8e4-74ea-4b83-8f1b-23c98ef2342f"}'

}
}

@notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"1f468faa-79ea-4076-a165-16bfe6909e80"}'

}

@notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"6f42eac1-93d7-4472-86c7-0b8b7fea1ce2"}'

}

@notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"2456b623-fd41-4b72-8b8f-27eb1b09006d"}'

}

[NOTION DEBUG - notion.pages.create-payload]: {

    @notionhq/client warn: request fail {

  code: 'conflict_error',

  message: 'Conflict occurred while saving. Please try again.'

}

Error adding quote to Notion: Error [APIResponseError]: Conflict occurred while saving. Please try again.

    at t.buildRequestError (.next/server/chunks/226.js:1:2950)

    at k.request (.next/server/chunks/226.js:1:19805)

    at async y.pages.create (.next/server/app/api/add-to-notion/route.js:40:7835)

    at async (.next/server/app/api/add-to-notion/quotes/route.js:1:2020)

    at async n (.next/server/app/api/add-to-notion/quotes/route.js:1:2217)

    at async v (.next/server/app/api/add-to-notion/route.js:40:2020) {

  code: 'conflict_error',

  status: 409,

  headers: [HeadersList],

  body: '{"object":"error","status":409,"code":"conflict_error","message":"Conflict occurred while saving. Please try again.","request_id":"43291e17-545c-4fe1-9b3c-1e6c568f73f0"}'

}